cli/conf.py

```diff
@@ -105,6 +123,81 @@ def parse_args(cls, module_name, config, args=None):
 
         return res_args
 
+
+class SpecLoader(object):
```

Comment:
Do we need a whole class of "class methods"? Seems like it would be simpler to make this class a different module (spec.py) with simple functions.